vector/src/main/AndroidManifest.xml

```diff
@@ -196,6 +198,30 @@
             </intent-filter>
         </activity>
 
+        <activity-alias
+            android:name=".features.permalink.PermalinkHandlerActivity"
+            android:exported="true"
+            android:launchMode="singleTask"
+            android:targetActivity=".features.link.LinkHandlerActivity">
```

Comment:
Can you add explicit `enable="true"` attribute? It will be nice for the forks who want to disable it.
Also maybe add a comment to explain which links type are handled here.